libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java

```diff
@@ -121,19 +120,21 @@ private GutenbergContainerFragment getGutenbergContainerFragment() {
     public Bundle getTranslations() {
         Bundle translations = new Bundle();
         Locale defaultLocale = new Locale("en");
-        Resources currentResources = getActivity().getApplicationContext().getResources();
-        Configuration currentConfiguration = currentResources.getConfiguration();
+        Resources currentResources = getActivity().getResources();
+        Context localizedContextCurrent = getActivity()
+                .createConfigurationContext(currentResources.getConfiguration());
         // if the current locale of the app is english stop here and return an empty map
+        Configuration currentConfiguration = localizedContextCurrent.getResources().getConfiguration();
```

Comment:
Wondering if we can use `currentResources.getConfiguration()` here. `currentResources` are read from the activity and should work.